pkg/engine/destroy.go

```diff
@@ -80,8 +80,7 @@ func newDestroySource(
 
 	// Like Update, if we're missing plugins, attempt to download the missing plugins.
 
-	if err := ensurePluginsAreInstalled(ctx, plugctx.Diag, plugins.Deduplicate(),
-		plugctx.Host.GetProjectPlugins()); err != nil {
+	if err := EnsurePluginsAreInstalled(ctx, plugctx.Diag, plugins.Deduplicate(), plugctx.Host.GetProjectPlugins(), false); err != nil {
```

Comment:
Lint errors due to the line being too long (several files). See https://github.com/pulumi/pulumi/actions/runs/10463830528/job/28976495359